### PR TITLE
Inertia switch

### PR DESCRIPTION
**Pull request by mflanag**
Made a redundant inertia switch that works by reading the IMU messages for acceleration. If it is on 16g resolution and reports over 8g then the Main will open the SDC. Unfortunately hardware people suck and main SDC doesn't actually work. I have tested it on the car and used a multimeter to confirm operation on the software side.

### CODE REVIEW

File: Inc/CANProcess.h [hunk 1]
```diff
@@ -15,9 +15,9 @@
 //defines for reading data from RxCanMsgTypeDef
 #define ID_PEDALBOX1							0x500
 #define ID_PEDALBOX2							0x501
-#define	ID_PEDALBOXCALIBRATE					0x503
```
Comment (xbenng) [inline Inc/CANProcess.h]: lets use astyle so we don't get stuff like this
http://astyle.sourceforge.net/

Comment (mflanag) [reply Inc/CANProcess.h]: i have implemented it in rinehart i think lol

**Comment by mflanag** [comment]
can someone approve this

**Review by bsmrdel — APPROVED** [review]
Looks good. Sorry, didn't realize this was waiting to be reviewed. Remember that eventually we'll need to verify the correct axis for the IMU since it's re-installed (and no longer under the seat). I believe @cfallon61 made these changes to reflect (x = forward, y = left, z = up), but once the car is working we'll need to confirm.

**Review by RyanSchw — APPROVED** [review]

**Pull request merged**